orchestration/dagster_orchestration/hca_orchestration/solids/copy_project/delete_outdated_tabular_data.py

```diff
@@ -4,37 +4,45 @@
 )
 from dagster_utils.contrib.data_repo.jobs import poll_job
 from dagster_utils.contrib.data_repo.typing import JobId
+from dagster_utils.contrib.google import path_has_any_data, parse_gs_path
 from data_repo_client import JobModel
 
 from hca_orchestration.contrib.bigquery import BigQueryService
-from hca_orchestration.resources.config.hca_dataset import TargetHcaDataset
-from hca_orchestration.resources.config.scratch import ScratchConfig
+from hca_orchestration.models.hca_dataset import HcaDataset
+from hca_orchestration.models.scratch import ScratchConfig
 
 
 @solid(
-    required_resource_keys={"bigquery_service", "target_hca_dataset", "scratch_config", "data_repo_client"},
+    required_resource_keys={"bigquery_service", "target_hca_dataset", "scratch_config", "data_repo_client", "gcs"},
     input_defs=[InputDefinition("entity_types", set[str])]
 )
 def delete_outdated_tabular_data(context: AbstractComputeExecutionContext, entity_types: set[str]) -> None:
     """Soft-deletes outdated and duplicate data in each entity type table"""
 
-    target_hca_dataset: TargetHcaDataset = context.resources.target_hca_dataset
+    target_hca_dataset: HcaDataset = context.resources.target_hca_dataset
     bigquery_service: BigQueryService = context.resources.bigquery_service
     scratch_config: ScratchConfig = context.resources.scratch_config
     data_repo_client = context.resources.data_repo_client
 
-    destination_path = f"{scratch_config.scratch_bucket_name}/{scratch_config.scratch_prefix_name}"
+    base_path = f"{scratch_config.scratch_bucket_name}/{scratch_config.scratch_prefix_name}"
     for entity_type in entity_types:
-        dupes_path = f"gs://{destination_path}/outdated_row_ids/{entity_type}/*"
-        _extract_dupes_to_scratch_area(bigquery_service, dupes_path, entity_type, target_hca_dataset)
+        destination_path = parse_gs_path(f"gs://{base_path}/outdated_row_ids/{entity_type}")
```

Comment:
All of this path manipulation makes me sad, I think a follow on here will to bring sanity to our path manipulation throughout HCA, but that's out of scope.